MAUI/DigitalGauge/Display-Types.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Display Types in Maui Digital Gauge control | Syncfusion
+description: Learn here all about Display Types support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+
+# Display Types in Maui Digital Gauge (SfDigitalGauge)
+
+The digital gauge displays numbers, alphabet, and special characters, which are given in the text property.
+
+## Setting value to number
+
+Numbers can be displayed in digital gauge in different formats using the `CharacterType`property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <gauge:SfDigitalGauge Text="12 34 56" StrokeWidth="5" 
+                          HeightRequest="100"  WidthRequest="300"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="LightSkyBlue"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SevenSegment" 
+                          CharacterStroke="#146CED"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 300;
+
+            this.BackgroundColor = Color.White;
+
+            digital.Text = "1 2 3 4 5";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 5;
+
+            digital.CharacterType = DigitalGaugeCharacterType.SevenSegment;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Setting value to alphabet
+
+Alphabet can be displayed in digital gauge using any one of the character format types.
```

Comment:
Alphabet can be displayed in a digital gauge using any one of the character format types.